### PR TITLE
umount_wipefs: add umount and wipefs steps

### PR DESCRIPTION
**Pull request by ybduan**
Add umount and wipefs steps before formatting in case the hard drive
contains a mounted filesystem.

ID: 1815053
Signed-off-by: ybduan <yduan@redhat.com>

### CODE REVIEW

**Comment by ybduan** [comment]
(1/7) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.usb.usb_storage.without_usb_hub.piix3-uhci: PASS (334.85 s)
 (2/7) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.usb.usb_storage.without_usb_hub.piix4-uhci: PASS (334.53 s)
 (3/7) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.usb.usb_storage.without_usb_hub.ich9-uhci: PASS (329.53 s)
 (4/7) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.usb.usb_storage.without_usb_hub.ich9-ehci: PASS (211.79 s)
 (5/7) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.usb.usb_storage.without_usb_hub.usb-ehci: PASS (211.51 s)
 (6/7) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.usb.usb_storage.without_usb_hub.nec-xhci: PASS (208.43 s)
 (7/7) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.usb.usb_storage.without_usb_hub.qemu-xhci: PASS (210.06 s)

**Comment by ybduan** [comment]
@XueqiangWei  Please help review this PR, thanks!

**Comment by XueqiangWei** [comment]
Linux guest works well.

 (1/7) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.usb.usb_storage.without_usb_hub.piix3-uhci: PASS (648.65 s)
 (2/7) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.usb.usb_storage.without_usb_hub.piix4-uhci: PASS (649.81 s)
 (3/7) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.usb.usb_storage.without_usb_hub.ich9-uhci: PASS (657.16 s)
 (4/7) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.usb.usb_storage.without_usb_hub.ich9-ehci: PASS (527.64 s)
 (5/7) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.usb.usb_storage.without_usb_hub.usb-ehci: PASS (512.00 s)
 (6/7) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.usb.usb_storage.without_usb_hub.nec-xhci: PASS (484.20 s)
 (7/7) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.usb.usb_storage.without_usb_hub.qemu-xhci: PASS (500.43 s)

**Comment by XueqiangWei** [comment]
Windows guest works well.

 (1/6) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.usb.usb_storage.without_usb_hub.piix3-uhci: PASS (224.68 s)
 (2/6) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.usb.usb_storage.without_usb_hub.ich9-uhci: PASS (218.70 s)
 (3/6) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.usb.usb_storage.without_usb_hub.ich9-ehci: PASS (173.80 s)
 (4/6) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.usb.usb_storage.without_usb_hub.usb-ehci: PASS (166.10 s)
 (5/6) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.usb.usb_storage.without_usb_hub.nec-xhci: PASS (158.50 s)
 (6/6) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.usb.usb_storage.without_usb_hub.qemu-xhci: PASS (156.65 s)

File: qemu/tests/usb_storage.py [hunk 1]
```diff
@@ -148,9 +135,6 @@ def _check_serial_option(serial, regex_str, expect_str):
         if serial not in ["EMPTY_STRING", "NO_EQUAL_STRING"]:
             # Verify in guest when serial is set to empty/null is meaningless.
             _verify_string(serial, output, [serial])
-        _check_and_umount_usb(session)
-        _do_io_test_guest(session)
```
Comment (XueqiangWei) [inline qemu/tests/usb_storage.py]: remove i/o test for check_serial_option, check_removable_option and check_io_size_option, so I suggest you add them to commits

**Comment by XueqiangWei** [comment]
@ybduan Do you have plan to add params \"rerror/werror\"?

File: qemu/tests/usb_storage.py (hunk 1 above)
Comment (ybduan) [reply qemu/tests/usb_storage.py]: Add the corresponding info to the comments. Thanks!

**Comment by ybduan** [comment]
> @ybduan Do you have plan to add params "rerror/werror"?

Because adding these new parameters need usb.usb_storage full testing and it takes time, I'll do that later. Please file a new automation bz for this requirement. Thanks!

**Comment by XueqiangWei** [comment]
LGTM,
Acked-by xuwei@redhat.com

**Comment by ybduan** [comment]
Please help review this PR, thanks! @lolyu

**Pull request merged**